change/@azure-msal-node-a018e8b0-79a2-4e4b-b407-4488edfb0010.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Test changes",
```

Comment:
Can you please provide a little more detail here? For example:

```suggestion
  "comment": "Tests for acquireTokenSilent #4672",
```

Reply:
Sounds good, will keep it in mind!